test/ProblemDetailsResponseFactoryFactoryTest.php

```diff
@@ -103,11 +104,7 @@ public function testUsesPrettyPrintFlagOnEnabledDebugMode() : void
         $factoryFactory = new ProblemDetailsResponseFactoryFactory();
         $factory = $factoryFactory($this->container->reveal());
 
-        $this->assertAttributeSame(
-            JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_PRESERVE_ZERO_FRACTION,
-            'jsonFlags',
-            $factory
-        );
+        $this->assertSame(JSON_PRETTY_PRINT, Assert::readAttribute($factory, 'jsonFlags') & JSON_PRETTY_PRINT);
```

Comment:
I always forget my bitwise operators - thanks for simplifying this!

Reply:
No worries. The test failed me when it really shouldn't so i thought it was a bit over-assertive. :)